README.md

```diff
@@ -451,6 +451,10 @@ Default: `thread`
 <dd>A list of `type` objects, (e.g. `type(my_class_instance)` or `MyClass`) that will be serialized using
     `repr()`. Default `[]`
 </dd>
+<dt>scrub_varargs
+</dt>
+<dd>If `True`, variable arguments will be scrubbed. Default `True`.
```

Comment:
I'm not 100% sure what you mean - since varargs are by definition unnamed, isn't the only option to scrub the values?
At any rate, here's the code that does it:
https://github.com/rollbar/pyrollbar/blob/41d96914906c382416c305a075da96b563521860/rollbar/__init__.py#L813